RPi-Pico/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.13)
 
 # Pull in Pico and FreeRTOS
 include(pico_sdk_import.cmake)
-include(pico_extras_import_optional.cmake)
+#include(pico_extras_import_optional.cmake)
```

Comment:
```diff --git a/RPi-Pico/CMakeLists.txt b/RPi-Pico/CMakeLists.txt
index 9af73bd..b3b2868 100644
--- a/RPi-Pico/CMakeLists.txt
+++ b/RPi-Pico/CMakeLists.txt
@@ -1,7 +1,8 @@
 cmake_minimum_required(VERSION 3.13)

 # Pull in Pico and FreeRTOS
-include(pico_sdk_import.cmake)
+set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}")
+include($ENV{PICO_SDK_PATH}/external/pico_sdk_import.cmake)
 #include(pico_extras_import_optional.cmake)
 #include($ENV{FREERTOS_KERNEL_PATH}/portable/ThirdParty/GCC/RP2040/FreeRTOS_Kernel_import.cmake)```

Reply:
The `set` was only needed for the other thing we tried, not needed for that final line, so I've only done the final one and updated the README accordingly.